src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog.jsx

```diff
@@ -385,8 +392,14 @@ const TwoWindingsTransformerModificationDialog = ({
             }
             const temporaryLimits2 = addModificationTypeToTemporaryLimits(
                 sanitizeLimitNames(limits[CURRENT_LIMITS_2]?.[TEMPORARY_LIMITS]),
-                twtToModify?.currentLimits2?.temporaryLimits,
-                editData?.currentLimits2?.temporaryLimits,
+                completeCurrentLimitsGroupsToOnlySelected(
+                    twtToModify?.currentLimits2,
+                    twtToModify?.selectedOperationalLimitsGroup2
+                )?.temporaryLimits,
+                completeCurrentLimitsGroupsToOnlySelected(
+                    editData?.currentLimits2,
+                    editData?.selectedOperationalLimitsGroup2
+                )?.temporaryLimits,
```

Comment:
I don't agree with the using of the addModificationTypeToTemporaryLimits function
 Try to simplify this code to avoid duplication and processing complexity
I suggest for example 
```suggestion
                 const { temporaryLimits1, temporaryLimits2 } = addModificationTypeToTemporaryLimits({
                limits: limits,
                currentLimitsData: lineToModify,
                editLimitsData: editData,
                currentNode,
            });
```

Reply:
I agree it could be improved but I didn't touch that code and line-modification isn't even the subject of this (already huge) PR.